cmd/goose/driver_turso.go

```diff
@@ -1,5 +1,5 @@
 package main
 
 import (
-	_ "github.com/libsql/libsql-client-go/libsql"
+	_ "github.com/tursodatabase/libsql-client-go/libsql"
```

Comment:
I presume the repository was transferred into the new organization, which is fine, matches up with https://github.com/tursodatabase/libsql-client-go/pull/94